io/net/virt-net/network_virtualization.py

```diff
@@ -172,6 +156,20 @@ def run_command(self, command, timeout=300):
         con.prompt(timeout)
         return output
 
+    def check_slot_availability(self):
+        '''
+        Checks if given slot is available.
+        Returns True if yes.
+        Returns False if not available
+        '''
+        cmd = 'lshwres -r virtualio -m %s --rsubtype vnic --filter \
+           "lpar_names=%s" -F slot_num' % (self.server, self.lpar)
+        for slot in self.run_command(cmd):
+            if int(self.slot_num) == int(slot):
+                self.log.debug("Slot already exists")
```

Comment:
is this condition should be
``` 
if int(self.slot_num) == int(slot):
     self.log.debug("Slot already exists")
        return true
return False```

Reply:
no, current condition is correct only

Reply:
ok than doc string need to be changed as when slot matched in should return false

Reply:
Docstring is also matching.

Slot availability = Slot does not exist already, so can be used.

Reply:
I will try and rephrase the docstring